pyvrp/tests/test_ProblemData.py

```diff
@@ -233,7 +261,7 @@ def test_problem_data_replace_with_changes():
     new = original.replace(
         clients=[Client(x=1, y=1)],
         vehicle_types=[VehicleType(3, 4), VehicleType(5, 6)],
-        distance_matrix=np.ones((2, 2), dtype=int),
+        distance_matrix=np.where(np.eye(2), 0, 2),  # All 2 except diagonal,
```

Comment:
```suggestion
        distance_matrix=np.where(np.eye(2), 0, 2),  # All 2 except diagonal
```
(since almost all the other changes of a similar form go undocumented, these comments might also just be superfluous and removed)